OpenRA.Mods.Common/Util.cs

```diff
@@ -84,16 +84,20 @@ public static WAngle AngleDiffToStep(WAngle facing, int numFrames)
 			return new WAngle(a % step - step / 2);
 		}
 
-		public static WAngle GetInterpolatedFacing(WAngle facing, int facings, int interpolatedFacings)
+		/// <summary>Returns the angle that the closest facing sprite should be rotated by to achieve the closest interpolated facing.</summary>
+		public static WAngle GetInterpolatedFacingRotation(WAngle facing, int facings, int interpolatedFacings)
 		{
+			if (facings == interpolatedFacings)
+				return WAngle.Zero;
```

Comment:
Currently it is impossible for facings to be = to interpolated Facings due to the below code in `DefaultSpriteSequence.cs`. I see you have kept this check in this overhaul as well, so we could save performance by omitting this if statement.

``` C#
				if (interpolatedFacings != -1 && (interpolatedFacings <= 1 || interpolatedFacings <= Math.Abs(facings) || interpolatedFacings > 1024
					|| !Exts.IsPowerOf2(interpolatedFacings)))
					throw new YamlException($"InterpolatedFacings must be greater than Facings, within the range of 2 to 1024, and a power of 2.");
```